eng/pipelines/templates/variables/vmr-build.yml

```diff
@@ -45,9 +45,8 @@ variables:
 - ${{ elseif or(startsWith(parameters.desiredIBC, 'Default'), eq(parameters.desiredIBC, '')) }}:
   # Enable IBC on the internal project if the, branch is a main or release branch
   - ${{ if and(eq(variables['System.TeamProject'], 'internal'), or(eq(variables['Build.SourceBranch'], 'refs/heads/main'), startsWith(variables['Build.SourceBranch'], 'refs/heads/internal/release/'), and(startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'), contains(variables['Build.SourceBranch'], '-preview')))) }}:
```

Comment:
according to the last condition we only enable this for `release/*-preview`, is that intentional?

Reply:
I think it should be release/10* let me update.

Reply:
why make it specific to 10?

Reply:
I guess this doesn't run for 9, so that seems reasonable.